libglusterfs/src/client_t.c

```diff
@@ -279,7 +279,7 @@ gf_client_ref(client_t *client)
     }
 
     GF_ATOMIC_INC(client->count);
-    gf_msg_callingfn("client_t", GF_LOG_DEBUG, 0, LG_MSG_REF_COUNT,
+    gf_msg_callingfn("client_t", GF_LOG_TRACE, 0, LG_MSG_REF_COUNT,
```

Comment:
Makes sense, but unrelated to the tiering feature?

Reply:
Strictly yes, it's unrelated, but we hit this more often with tiering because the operation of the tiering 'engine' (the thing running the migrate-to-cold script) makes this trigger often.

Reply:
> Strictly yes, it's unrelated, but we hit this more often with tiering because the operation of the tiering 'engine' (the thing running the migrate-to-cold script) makes this trigger often.

I understand. I do believe you should submit this in a separate PR, which I assume would be easier (if accepted!) to get in, then you can rebase on that. Right now it just makes your existing PR harder to review.